cluster/cluster_database.proto

```diff
@@ -17,36 +17,39 @@
 
 syntax = "proto3";
 
-option java_package = "grakn.protocol.cluster";
-option java_outer_classname = "DatabaseProto";
+option java_package = "grakn.protocol";
+option java_outer_classname = "ClusterDatabaseProto";
 
-package grakn.protocol.cluster;
+package grakn.protocol;
 
-message Database {
-
-    string name = 1;
-    repeated Replica replicas = 2;
-
-    message Replica {
-        string address = 1;
-        bool primary = 2;
-        bool preferred = 3;
-        int64 term = 4;
-    }
+message ClusterDatabaseManager {
 
     message Get {
         message Req {
             string name = 1;
         }
         message Res {
-            Database database = 1;
+            ClusterDatabase database = 1;
         }
     }
 
     message All {
         message Req {}
         message Res {
-            repeated Database databases = 1;
+            repeated ClusterDatabase databases = 1;
         }
     }
 }
+
+message ClusterDatabase {
+
+    string name = 1;
+    repeated Replica replicas = 2;
+
+    message Replica {
+        string address = 1;
+        bool primary = 2;
+        bool preferred = 3;
+        int64 term = 4;
+    }
+}
```

Comment:
There's a missing newline but I think it's fine right? Newline is not required, I've never seen it cause a build to be broken

Reply:
We should always end a file with a newline regardless of build not complaining, @lolski , so I'll update this